_posts/2016-08-02-omero-5-2-5.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Security release of OMERO 5.2.5
+intro-blurb: The OME team are pleased to announce the release of OMERO 5.2.5
+---
+Today we are releasing OMERO 5.2.5. This is a security release to fix
+the OMERO.shares API which was bypassing intended security
+restrictions and granting elevated privileges to read
+otherwise-restricted data.
+
+Full details of the issue are available
+[here](https://www.openmicroscopy.org/site/products/omero/secvuln/2016-SV2-share).
+
+This release **does not** upgrade the version of Bio-Formats which
+OMERO uses.
+
+The software is available [here](http://downloads.openmicroscopy.org/omero/5.2.5).
+
+Upgrade information is
+[here](http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-upgrade.html).
```

Comment:
Replace by
``Upgrade information is in the [server upgrade documentation](url)``